src/pages/programs/program-card.jsx

```diff
@@ -53,7 +53,7 @@ export const ProgramCard = ({
                   <img
                     src={pageData.externalIcon}
                     alt="external-link-icon"
-                    style={{ width: "20px" }}
+                    style={{ width: '20px' }}
```

Comment:
let us still this using `@emotion`. We should try to avoid inline styles.

Reply:
to keep it consistent, styled is used

Reply:
replaced by @emotion styled.